README.md

```diff
@@ -22,10 +22,20 @@ const MyComponent = () => <CloseSVG />;
 
 ## Installation
 
+You only need to install this package as a dev dependency because it's only needed during compilation.
```

Comment:
is this not implied, since babel will already exist in the appropriate category?

separately, this has to be a runtime dep if it's used in a babel preset - i'm not sure this is helpful advice here.